.github/workflows/bump.yaml

```diff
@@ -0,0 +1,31 @@
+name: Publish packages on NPM
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'What to bump?'
+        required: true
+        default: 'patch'
+        type: choice
+        options:
+        - 'patch'
+        - 'minor'
+        - 'major'
+
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version-file: 'package.json'
+
+    - run: yarn version:all ${{ inputs.version }}
+    - run: git add .
+    - run: VERSION=`yarn workspace @datadog/webpack-plugin info @datadog/webpack-plugin --json | jq -r '.children.Version'`
+    - run: TAG_NAME="v$VERSION"
+    - run: git commit -m $TAG_NAME
+    - run: git tag -a $TAG_NAME -m $TAG_NAME
+    - run: git push --follow-tags
```

Comment:
I think this might not work without a proper token.
But I need to merge this first in order to be able to actually debug it.

Reply:
> But I need to merge this first in order to be able to actually debug it.

FWIW it is possible to trigger a workflow dispatch from a branch using the gh tool (`gh workflow run workflow-name --ref branch-name`)

Reply:
😞 

```bash
╰─❯ gh workflow run bump.yaml --ref origin/yoann/release-process-update -f version=patch

HTTP 404: Not Found (https://api.github.com/repos/DataDog/build-plugins/actions/workflows/bump.yaml)
```